styles/header.css

```diff
@@ -132,3 +132,9 @@
   margin: 1rem;
   background-color: var(--color-grey-dark-4);
 }
+
+@media screen and (max-width: 550px) {
```

Comment:
use rem here as well for max-width

Reply:
foe media query, I want to fix the size related to the view port.